### PR TITLE
fix 2 broken links in dm docs

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Fix two broken links found in https://github.com/pingcap/docs-cn/issues/8928

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-from-sql-files-to-tidb.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ file = "tidb-lightning.log"
 [tikv-importer]
 # "local"：默认使用该模式，适用于 TiB 级以上大数据量，但导入期间下游 TiDB 无法对外提供服务。
 backend = "local"
-# # "tidb"：TiB 级以下数据量也可以采用 `tidb` 后端模式，下游 TiDB 可正常提供服务。关于后端模式更多信息请参 https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends。
+# # "tidb"：TiB 级以下数据量也可以采用 `tidb` 后端模式，下游 TiDB 可正常提供服务。关于后端模式更多信息请参考 https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends 。
```
Comment (shichun-0415) [inline migrate-from-sql-files-to-tidb.md:50]:
```suggestion
# # "tidb"：TiB 级以下数据量也可以采用 `tidb` 后端模式，下游 TiDB 可正常提供服务。关于后端模式更多信息请参考 [TiDB Lightning 后端](/tidb-lightning/tidb-lightning-backends.md) 。
```

Comment (shichun-0415) [inline migrate-from-sql-files-to-tidb.md:50]:
```suggestion
# # "tidb"：TiB 级以下数据量也可以采用 `tidb` 后端模式，下游 TiDB 可正常提供服务。关于后端模式更多信息请参考 [TiDB Lightning 后端](/tidb-lightning/tidb-lightning-backends.md) 。
```

File: dm/shard-merge-best-practices.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ CREATE TABLE `tbl_multi_pk` (
 
 ## 上游 RDS 封装分库分表的处理
 
-上游数据源为 RDS 且使用了其分库分表功能的情况下，MySQL binlog 中的表名在 SQL client 连接时可能并不可见。例如在 UCloud 分布式数据库 [UDDB](https://www.ucloud.cn/site/product/uddb.html) 中，其 binlog 表名可能会多出 `_0001` 的后缀。这需要根据 binlog 中的表名规律，而不是 SQL client 所见的表名，来配置 [table routing 规则](/dm/dm-key-features.md#table-routing)。
+上游数据源为 RDS 且使用了其分库分表功能的情况下，MySQL binlog 中的表名在 SQL client 连接时可能并不可见。例如在 UCloud 分布式数据库 [UDDB](https://docs.ucloud.cn/uddb/README) 中，其 binlog 表名可能会多出 `_0001` 的后缀。这需要根据 binlog 中的表名规律，而不是 SQL client 所见的表名，来配置 [table routing 规则](/dm/dm-key-features.md#table-routing)。
```
Comment (shichun-0415) [inline dm/shard-merge-best-practices.md:120]: 这个二级标题和内容在 docs 仓库没见着，请帮忙看看是不是漏译了。

File: migrate-from-sql-files-to-tidb.md (hunk 1 above)
Comment (ran-huang) [reply migrate-from-sql-files-to-tidb.md:50]: 在注释里的链接不建议写成 Markdown link 格式，用户不能直接点击访问，也不能复制到地址栏访问

Comment (shichun-0415) [inline migrate-from-sql-files-to-tidb.md:50]:
```suggestion
# # "tidb"：TiB 级以下数据量也可以采用 `tidb` 后端模式，下游 TiDB 可正常提供服务。关于后端模式更多信息请参考 https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-backends。
```

File: dm/shard-merge-best-practices.md (hunk 2 above)
Comment (ran-huang) [reply dm/shard-merge-best-practices.md:120]: 等会儿补上

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**